tendrl/gluster_integration/sds_sync/brick_utilization.py

```diff
@@ -59,10 +59,12 @@ def get_lvs():
             {"message": str(err)}
         )
         return None
-    out = out.split('\n')
-    lst = map(lambda x: dict(x),
-              map(lambda x: [e.split('=') for e in x],
-                  map(lambda x: x.strip().split('$'), out)))
+    lst = {}
```

Comment:
We are calling get_lvs()  function in each gluster to collect the type lvm block device details. If a storage node which does not have any lvm type block device then parsing logic is failing. And that exception is caught by the main try-catch block which means it will skip all other function calls and try to execute the sync from starting. 

So node's first sync never marked as done, so import will timeout.


Here i am checking out is empty or not if it is empty then i am skipping parsing logic.

Reply:
ok